apis/beacon/states/fork.yaml

```diff
@@ -0,0 +1,24 @@
+get:
+  operationId: "getStateFork"
+  summary: "Get Fork object for requested state"
+  description: "Returns [Fork](https://github.com/ethereum/eth2.0-specs/blob/v0.11.1/specs/phase0/beacon-chain.md#fork) object for state with given 'stateId'."
```

Comment:
Do we intend to hard-code `v0.11.1` for spec references? Will we update all after this initial release?

Reply:
I'm keeping them all the same, so I can FindAndReplace to update everything to 0.12 once we have all the endpoints :smile: